src/api/axiosInstanceFactory.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { HTTP_STATUS, HEADERS } from '../constants/http.constants';
+
+export const createAxiosInstance = ({ getToken, onUnauthorized }) => {
+  const instance = axios.create({
+    baseURL: import.meta.env.VITE_API_URL,
+    withCredentials: true,
```

Comment:
Maybe instead of `VITE_API_URL` call this `BASE_SERVER_URL` or anything that indicates what the variable is doing. 

Also please update the example env to include the new key